docs/sdks/client-sdks/javascript/Initialization.mdx

```diff
@@ -0,0 +1,296 @@
+---
+title: Initialization
+sidebar_position: 3
+---
+
+import ApiOptionRef from '@site/src/components/ApiOptionRef';
+
+
+The Eppo JavaScript SDK is simple to initialize but provides a number of options for customization that makes it flexible for a wide range of use cases including offline mode, custom caching requirements, and extremely low latency initialization.
+
+## Initialize the SDK
+
+To complete basic initialization, you only need to provide your SDK key which you can find in the Eppo UI under **Admin** > **API Keys**. 
+
+```js
+const eppo = new Eppo({
+  apiKey: 'YOUR_SDK_KEY',
+});
+``` 
+
+## Use the SDK instance
+After initialization, you can use the SDK instance by importing the SDK at the scope you will use it in and calling `getInstance()`. You can then use the SDK instance to assign a variation to a subject using the `get*Assignment` functions.
+
+```javascript
+import * as EppoSdk from "@eppo/js-client-sdk";
+
+const eppoClient = EppoSdk.getInstance();
+const user = getCurrentUser();
+
+```
+
+
+## Advanced Configuration
+Basic initialization is great for most use cases, but the SDK provides options that you can use during initialization to customize the behavior of the SDK.
+
+### Initialization Options
+
+How the SDK fetches, serves, and caches experiment configurations is configurable via additional optional initialization options:
+
+<ApiOptionRef 
```

Comment:
These look a bit awkward when not in dark mode. We should drop the outline over the `name`

![image](https://github.com/user-attachments/assets/613e9e00-39a2-41cf-814d-12f0bf679c24)